docs/troubleshooting.asciidoc

```diff
@@ -0,0 +1,236 @@
+[id="{p}-troubleshooting"]
+== Troubleshoot a failing cluster
+
+When things don't seem to work as expected, there are several places you can look at to investigate.
```

Comment:
When things don't work as expected, you can investigate by taking the following actions:

- <<{p}-get-resources,Get the list of resources>>
- <<{p}-describe-failing-resources,Describe failing resources>>
- <<{p}-get-elasticsearch-logs,Get Elasticsearch logs>>
- <<{p}-get-init-container-logs,Get init container logs>>
- <<{p}-get-eck-logs,Get ECK logs>>
- <<{p}-eck-debug-logs,Enable ECK debug logs>>
- <<{p}-pause-controllers,Pause ECK controllers>>
- <<{p}-get-k8s-events,Get Kubernetes events>>
- <<{p}-exec-into-containers,Exec into containers>>
- <<{p}-ask-for-help,Ask for help>>

Reply:
As the section is relatively long, it may help to list the subsections here.